Library/Formula/xmlto.rb

```diff
@@ -40,10 +40,10 @@ def install
 +++ xmlto-0.0.25/xmlto.in
 @@ -209,7 +209,7 @@
  export VERBOSE
- 
+
  # Disable network entities
 -XSLTOPTS="$XSLTOPTS --nonet"
 +#XSLTOPTS="$XSLTOPTS --nonet"
- 
+
```

Comment:
This patch does not seem to work anymore and he induced me to find a workaround in #48935. The patched file seems to be modified with no version change, and this seems to break the patch. The bottled version apparently contains the broken patch and fails to let xslproc to download the needed DTDs. Should we replace this patch with an `inreplace` substitution so we can make sure the amendment is done regardless of source file changes? The upstream seems to be intent on keeping the `nonet` option as a default as a 'security feature' or something in this order.